webpush/encrypt_test.go

```diff
@@ -169,31 +185,44 @@ func rfcKeys() ([]byte, []byte, error) {
 	return priv, elliptic.Marshal(curve, x, y), nil
 }
 
-// TestRfcVectors uses the values given in the RFC for HTTP encryption to verify
+func rfcAes128gcmSalt() ([]byte, error) {
```

Comment:
I'd say `AES` is an initialism: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
Here and everywhere else.